cloud_filter_parameters:
  min_x: -0.5
  max_x: 1.0
  min_y: -0.6
  max_y: 0.6
  min_z: -0.5
  max_z: 1.0
